cmd/service.go

```diff
@@ -102,7 +102,12 @@ func (m *ServerMux) handleServiceSignals() error {
 			}
 			runExitFn(nil)
 		case serviceStop:
-			log.Println("Gracefully stopping... (press Ctrl+C again to force)")
+			// Extra space will prevent showing "^CExiting" and rather show "^C Exiting" instead.
+			log.Println(" Exiting")
+			go func() {
+				time.Sleep(time.Millisecond * 600)
+				log.Println("Waiting for active connections to terminate, press ^C again to exit immediately.")
```

Comment:
There appears to be another case - the current signal handling setup does not distinguish between a keyboard interrupt and being sent SIGTERM - they are not the same thing. It's actually incorrect to say  `press ^C again`, when the user may not have pressed `^C` in the first place. Also `^C` is not a widely intelligible term - it should be `Ctrl+C`.